lib/init.php

```diff
@@ -115,52 +124,55 @@ function beans_add_theme_support() {
 	// Beans specific.
 	add_theme_support( 'offcanvas-menu' );
 	add_theme_support( 'beans-default-styling' );
-
 }
 
 add_action( 'beans_init', 'beans_includes' );
 /**
  * Include framework files.
  *
+ * @since 1.0.0
  * @ignore
+ * @access private
+ *
+ * @return void
  */
 function beans_includes() {
 
 	// Include admin.
 	if ( is_admin() ) {
 
-		require_once( BEANS_ADMIN_PATH . 'options.php' );
-		require_once( BEANS_ADMIN_PATH . 'updater.php' );
-
+		require_once BEANS_ADMIN_PATH . 'options.php';
+		require_once BEANS_ADMIN_PATH . 'updater.php';
 	}
 
 	// Include assets.
-	require_once( BEANS_ASSETS_PATH . 'assets.php' );
+	require_once BEANS_ASSETS_PATH . 'assets.php';
 
 	// Include customizer.
 	if ( is_customize_preview() ) {
-		require_once( BEANS_ADMIN_PATH . 'wp-customize.php' );
+		require_once BEANS_ADMIN_PATH . 'wp-customize.php';
 	}
 
 	// Include renderers.
-	require_once( BEANS_RENDER_PATH . 'template-parts.php' );
-	require_once( BEANS_RENDER_PATH . 'fragments.php' );
-	require_once( BEANS_RENDER_PATH . 'widget-area.php' );
-	require_once( BEANS_RENDER_PATH . 'walker.php' );
-	require_once( BEANS_RENDER_PATH . 'menu.php' );
-
+	require_once BEANS_RENDER_PATH . 'template-parts.php';
+	require_once BEANS_RENDER_PATH . 'fragments.php';
+	require_once BEANS_RENDER_PATH . 'widget-area.php';
+	require_once BEANS_RENDER_PATH . 'walker.php';
+	require_once BEANS_RENDER_PATH . 'menu.php';
 }
 
 add_action( 'beans_init', 'beans_load_textdomain' );
 /**
  * Load text domain.
  *
+ * @since 1.0.0
  * @ignore
+ * @access private
```

Comment:
The access is not private.  Go ahead and remove the `@access` tag.  We do want to ignore it though for the automatic doc generator.